.jenkins/pytorch/test.sh

```diff
@@ -300,7 +300,7 @@ test_backward_compatibility() {
   pushd test/backward_compatibility
   python -m venv venv
   . venv/bin/activate
-  pip_install --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
+  pip_install --pre torch -f https://download.pytorch.org/whl/test/cpu/torch_test.html
```

Comment:
are we planning to use this in the future for RCs?

Reply:
This change is only specific for `release/1.8` so yes we’re planning on using this in the future for RCs